README.md

```diff
@@ -17,6 +19,45 @@ After that, clone the repo and inside any repo directory run:
 
 This will build and run a Rinvid test example. If it succeeds, you are all set.  
 
+### Windows 10
+
+What you need to build Rinvid on Windows 10:
+
+   1. [MinGW](http://mingw-w64.org/doku.php)  
+   2. [Bazel](https://bazel.build/)  
+   3. [MSYS](https://www.msys2.org/)
+   3. [SFML](https://www.sfml-dev.org/)  
+   4. [CMake](https://cmake.org/download/) (needed only if you're gonna build SFML from source)  
+
+#### Step 1
+Install MinGW from this [link](http://mingw-w64.org/doku.php). Make sure the g++ compiler is selected when choosing which packages to install in MinGW installation manager. Note: if you don't want to build SFML from source, visit [this page](https://www.sfml-dev.org/download/sfml/2.5.1/) first and check which exact version of MinGW you need to install in order for SFML to work correctly.
+
+#### Step 2
+Install Bazel by following instructions from this [link](https://docs.bazel.build/versions/master/install-windows.html)
+
+#### Step 3
+Install MSYS2 by follwing instructions from this [link](https://www.msys2.org/)  
+
+#### Step 4
+Copy contents of folder where your MinGW installed into <msys2_install_path>/mingw64  
+
+#### Step 5
+Add <msys2_install_path>/mingw64/bin to Path environment variable, and remove original MinGW installation location from Path env var.  
+
+#### Step 6 (Only if building SFML from source)
+Install CMake from this [link](https://cmake.org/download/)  
+
+#### Step 7
+Download SFML from this [page](https://www.sfml-dev.org/download/sfml/2.5.1/). If you're building SFML from source, follow instructions from this [link](https://www.sfml-dev.org/tutorials/2.5/compile-with-cmake.php)
```

Comment:
Seems like SFML/Window.hpp can't be found. Are we sure this is right location for SFML headers?

Reply:
Apparently, location of standard compiler includes can vary. I'll update the manual.

Reply:
Yeah. Seems like it's enough to find which `stdio` was being loaded and just place SFML folder there. 

Also seems like windows needs to be in "developer mode".